src/packages.ts

```diff
@@ -94,13 +86,27 @@ export class PackageManager {
 
         pkg.tmpFile = tmpResult;
 
-        const result = await downloadFile(url, pkg, logger, status, proxy);
+        const result = await downloadFile(latestSnootyReleaseUrl, pkg, logger, status, proxy);
         logger.appendLine(' Done!');
 
         return result;
     }
 }
 
+async function getLatestParserUrl (platform: string, logger: Logger) {
+    const githubUrl = `https://github.com/mongodb/snooty-parser/releases/latest`;
+    const response = await fetch(githubUrl, { headers: { Accept: 'application/json' } });
+    const json = await response.json() as { [k:string]: string };
+
+    let latestTag = json?.tag_name;
+    if (!latestTag) {
+        logger.appendLine('Error accessing newest snooty-parser version.');
+        latestTag = 'v0.14.10';
+    }
+    const latestParserReleaseUrl = `https://github.com/mongodb/snooty-parser/releases/download/${latestTag}/snooty-${latestTag}-${platform}_x86_64.zip`;
```

Comment:
I think we might want to include the architecture as well, but I am not sure it's an issue since we only use `x86-64` for the architecture previously. We would also need to upload the ARM version of the parser as well for the release, which we currently don't do.

Reply:
Yeah I was thinking about this. Honestly I think hard-coding x86_64 is okay *for now* since we have no even distant road map for doing native ARM64 builds, and given the caveats and pricing I'm seeing for running CI on ARM64, I don't see that changing any time soon :(